resources/files/windows/run_facter_interactive.bat

```diff
@@ -1,5 +1,2 @@
-@echo Running puppet on demand ...
-@echo off
-SETLOCAL

```

Comment:
Only because the original file run_facter_interactive did not have them but run_puppet_interactive did. 

The only reason was to be the same as the originals